app/plans/index.njk

```diff
@@ -8,58 +8,55 @@
 {% endblock %}
```

Comment:
NB this file changed just to fix a minor layout issue and subsquent indenting. No need to scrutinise too closely!